src/main/java/com/sericulture/authentication/controller/UserInfoController.java

```diff
@@ -118,7 +118,7 @@ else if(!encoder.matches(authRequest.getPassword(),userInfo.get().getPassword())
             loginApiResponse.setFirstName(userInfo.get().getFirstName());
             loginApiResponse.setLastName(userInfo.get().getLastName());
             loginApiResponse.setEmailId(userInfo.get().getEmailID());
-            loginApiResponse.setRoleId(userInfo.get().getRoleId());
+            loginApiResponse.setRoleId(userInfo.get().getRole().getRoleId());
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'else' child has incorrect indentation level 12, expected level should be 6.